src/nvim/options.lua

```diff
@@ -666,6 +666,14 @@ return {
       varname='p_emoji',
       defaults={if_true={vi=true}}
     },
+    {
+      full_name='encodefunc', abbreviation='encfunc',
```

Comment:
I would really suggest to discard abbreviation. Abbreviation dropping 30% of characters (3 out of 10) is not very useful and abbreviation like `enf` is cryptic. It is not like this option is going to be commonly set manually.